bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieProtocol.java

```diff
@@ -47,6 +47,11 @@
      */
     byte CURRENT_PROTOCOL_VERSION = 2;
 
+    /**
```

Comment:
This file is only for v2 protocol. I would suggest not touching this file. You can use `ProtocolVersion` instead. `ProtocolVersion` already defines the version numbers from 1 to 3.

Reply:
Sure